examples/process_gain.py

```diff
@@ -82,9 +82,7 @@ def determine_power_from_fit(filename, guessamp, guessph):
         )
         # {{{ Fit to complex
         d = psd.lmfitdata(d)
-        d.functional_form = A * sp.exp(
-            1j * 2 * pi * omega * t + 1j * phi
-        )
+        d.functional_form = A * sp.exp(1j * 2 * pi * omega * t + 1j * phi)
```

Comment:
seems to just be a black issue (probably forgot to change line length to 79 after editing the scripts in the manuscript to be 58)

Reply:
yes, but I had thought this was b/c of omega/nu again -- I'm going to fix and push that.